File: Python/Class.py
class Test():
	#consttructor
	def __init__(self,a,b):
		self.a=a
		self.b=b

	def add(self):
		c = a+b
		print(c)
	def mul(self):
		print(a*b)

	#destructor
	def __del__(self):
		print("Destructor")

if __name__ == "__main__":

	a = int(input())
	b = int(input())
	#create class object
	test = Test(a,b)
	test.add()
	test.mul()

		

File: C++/Class.cpp
#include<iostream>
using namespace std;

class Test{
	private: int a,b,c;
	public: Test(int a, int b){
        this->a = a;
        this->b = b;
	}
	public: int add(){
		c = a+b;
		cout<<c;
	}
	public: int mul(){
        cout<<a*b;
	}
	~Test(){
		cout<<"Destructor";
	}
};

int main(){
    int a,b;
	cin>>a>>b;
	Test test(a,b);
	test.add();
	test.mul();
	return 0;
}
